specification/eventgrid/Azure.Messaging.EventGrid.SystemEvents/Microsoft.Communication/AzureCommunicationServices.tsp

```diff
@@ -692,78 +692,78 @@ model AcsRouterWorkerSelector {
   key?: string;
 
   /** Router Job Worker Selector Label Operator */
-  operator?: AcsRouterLabelOperator;
+  operator: AcsRouterLabelOperator;
```

Comment:
Is this actually supposed to deserialize from 'labelOperator', not 'operator'?

(https://learn.microsoft.com/en-us/azure/event-grid/communication-services-router-events#microsoftcommunicationrouterjobqueued)